crypto/s2n_hmac.c

```diff
@@ -252,14 +265,18 @@ int s2n_hmac_update(struct s2n_hmac_state *state, const void *in, uint32_t size)
      * input. On some platforms, including Intel, the operation can take a
      * smaller number of cycles if the input is "small".
      */
-    state->currently_in_hash_block += (4294949760 + size) % state->hash_block_size;
+    const uint32_t HIGHEST_32_BIT = 4294949760;
```

Comment:
This name is confusing.

Reply:
This trigger will trigger changes on SAW and SideTrail pacthes, so I'll tackle it on a separate PR. I opened issue https://github.com/awslabs/s2n/issues/2521 to track this request.